server/gokbg3/src/integration-test/groovy/gokbg3/rest/PackageTestSpec.groovy

```diff
@@ -171,6 +171,7 @@ class PackageTestSpec extends AbstractAuthSpec {
       descriptionURL : "https://heise.de",
       fixed          : "Yes",
       global         : "Consortium",
+      globalNote     : "Testing Consortium",
```

Comment:
there's no testing for the new property here. 
add something like     resp.json.globaleNote == "Testing Consortium" in the then: section of the test method

Reply:
Added the test results